config/mongo.js

```diff
@@ -1,5 +1,7 @@
 import mongoose from "mongoose";
 
-const connectMongo = async () => mongoose.connect(process.env.MONGODB_URI);
+const connectMongo = async () => mongoose.connect(process.env.MONGODB_URI).then((data) => console.log("Hello connected")).catch((res) => console.log("Here is error"));
+
```

Comment:
As the function is using `async`, it might be better to use `await` with a `try/catch`